src/sage/numerical/interactive_simplex_method.py

```diff
@@ -3904,8 +3904,36 @@ def __init__(self, A, b, c, objective_value,
         c = copy(c)
         B = vector(basic_variables)
         N = vector(nonbasic_variables)
+        # Sadly, vector does not guarantee that the result is freshly allocated
+        # if the input was already a vector: #29101
+        if B is basic_variables:
```

Comment:
Please format it as `Issue #29101`. Please remove `Sadly`, which is inappropriate in code comment.

Reply:
Done in dd5269d5580